website/docs/r/api_management.html.markdown

```diff
@@ -163,6 +165,12 @@ A `proxy` block supports the following:
 
 ---
 
+A `protocols` block supports the following:
+
+* `enable_http2` - (Optional) Should HTTP/2 support be enabled on the client-facing side of the gateway?
```

Comment:
If we do add `Default: false` we should make a note of it here as well that it defaults to false.

Reply:
Added Defaults to `false` in documentation.